packages/amplify-category-geo/amplify-plugin.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "geo",
   "type": "category",
-  "commands": ["add", "push", "remove", "update", "help", "console"],
+  "commands": ["add", "push", "remove", "update", "help", "console", "populate"],
```

Comment:
This is a big PR. Has this been reviewed in depth with a few members of the CLI team? Has this `populate` command and the related UI flows gone through a full API review?